casr/src/bin/casr-libfuzzer.rs

```diff
@@ -81,6 +80,12 @@ fn main() -> Result<()> {
                 .long("no-cluster")
                 .help("Do not cluster CASR reports")
         )
+        .arg(
+            Arg::new("casr-gdb-args")
+                .long("casr-gdb-args")
+                .action(ArgAction::Set)
+                .help("Specify casr-gdb target arguments to add casr reports for non-instrumented binary"),
```

Comment:
Maybe, we should write similar help message as in casr-afl

Reply:
done here and below